docs/tuning.md

```diff
@@ -264,12 +264,16 @@ parent RDD's number of partitions. You can pass the level of parallelism as a se
 or set the config property `spark.default.parallelism` to change the default.
 In general, we recommend 2-3 tasks per CPU core in your cluster.
 
+# Parallel Listing on Input Paths
```

Comment:
This should be second-level title? i.e., ## Parallel Listing on Input Paths

Reply:
Good catch. Fixing.